### DISCUSSION TITLE
FR: only walls in extruder offset

### DISCUSSION THREAD

**Discussion by rubienr** [5axes/Calibration-Shapes#145] [Ideas]
Hi @5axes, 

I find your Extruder-Offset calibration pattern very helpful. Its awesome! Great job! I know it has been a while from the initial feature request until today, but finally I had time to test it :)

I found that the printing time can be reduced to about two minutes (without heating; from brim till end) by printing only the walls.
In my case i used the Cura setting "Surface Mode" (Special Modes -> Surface Mode -> Surface) to slice only walls and the result was perfect. Probably you could mention this in the [description page](https://github.com/5axes/Calibration-Shapes/wiki/ExtruderOffsetCalibration). Alternatively the model could be updated.

**Results** 

[1] first attempt with standard slicer settings
![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/3d5658ae-3382-4855-aefe-a02a38731f8f)

[2] reduced printing time to 1/3 w.r.t [1] (Cura -> Special Modes -> Surface Mode -> Surface)
![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/bb518140-ecd4-4409-97e4-59a8d5b1fe3d)

[3] even shorter print time (smaller wipe tower diameter, about 2 minutes)
![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/67ffbe9e-cd3f-4ce2-ae1e-164cf3c3fed6)

![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/75b860f5-0d48-45bd-b5dd-2266c15e3d27)

**Suggestion**

Pattern could consist only of walls (one, maximum two)  and also the center markers may be realized in the first layer. (_Off-topic: I found that switching one nozzle to another after the very first layer takes extremely long w.r.t to any other subsequent switch after the layer finished. This is very annoying and I suspect something wrong with Cura's generated code because tools already had the required temperature... This is why a second layer hurts me so much._).

0.4mm wall, 0.2mm layer height, line width 100%, flow rate 100%, no bed leveling but slightly cleaned with Isopropyl alcohol (very good adhesion):
![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/0c343ad7-bf57-43bd-8d63-507edf0b215d)

Apart of that It would be nice to connect all combs so they can be pulled off the bed at once :) 
I'd like to avoid touching the bed too often after it was cleaned with alcohol. 

![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/bc043017-8d42-4121-a306-944418b3f716)

At the time of writing i found that skirt with 0mm distance could achieve same effect. This pattern was printed in 120 seconds (first layer):

![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/48aa1b26-4421-4e30-807c-857552f3d72d)
![image](https://github.com/5axes/Calibration-Shapes/assets/7566204/7ce98a89-1afc-4007-9706-782cdf4be627)

Probably its worth to note it in the [description page](https://github.com/5axes/Calibration-Shapes/wiki/ExtruderOffsetCalibration) as well. Unfortunately Cura still has lots of bugs in multi extruder mode, and one is that the wipe tower cannot be placed anywhere. Sometimes it leads to funny slicing results where for example not all objects are sliced. In this particular case it is not possible to place the wipe tower in the inner corner.

**Comment by rubienr** [comment]
One more thing :)
Can we have a 2nd pattern with 0.1mm spacing? Same comb size, same teeth size/amount but 0.1mm offset.

**Comment by 5axes** [comment]
Post moved to the discussion section.
Wiki page updated : https://github.com/5axes/Calibration-Shapes/wiki/ExtruderOffsetCalibration

Thank you for this suggestion.

**Comment by rubienr** [comment]
I will test if it works with skirt distance 0.2 mm. From my point of view it should touch the comb w. 0mm distance.

**Reply by rubienr** [reply]
I found that it works best for me with with 0mm skirt distance. By that the chance is higher that I can peel everything off at once.

**Reply by 5axes** [reply]
Ok I Will update the Wiki . Thanks for your feedback, always much appreciated.

**Reply by rubienr** [reply]
You are welcome but: thank you for implementation and maintenance! :)
